connectors/javax.websocket-client/build.gradle

```diff
@@ -11,8 +11,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-archivesBaseName = project.name
```

Comment:
Good day! @bjhargrave 
Could I kindly ask this property(archivesBaseName) is used?